README.md

```diff
@@ -18,3 +18,10 @@ Accessible web components for the modern web.
 ### Installation
 
 `npm i @travelopia/web-components`
+
+### Development
+
+* Pull request are welcomed.
+* To start the development server and work locally run: `php -S localhost:3000`.
```

Comment:
Start the environment with a local server of your choice, for example: `php -S localhost:3000`